website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -66,7 +63,6 @@ func up() {
             "initialize",
             {"new_authority": 123},
             config={
-                "auto_estimate_fee": True,
                 "wait_for_acceptance": True,
```

Comment:
I think this config should include `max_fee`.

Reply:
[fix docs](https://github.com/software-mansion/protostar/pull/979/commits/794612a37db566139eaf1c36bad0577a0d6e83fc)